extensionsGallery.json

```diff
@@ -63,6 +63,59 @@
 					"statistics": [],
 					"flags": "preview"
 				},
+				{
+					"extensionId": "10",
+					"extensionName": "vscodeintellicode",
+					"displayName": "Visual Studio IntelliCode",
+					"shortDescription": "AI-assisted development",
+					"publisher": {
+						"displayName": "Microsoft",
+						"publisherId": "Microsoft",
+						"publisherName": "Microsoft"
+					},
+					"versions": [
+						{
+							"version": "1.2.1",
+							"lastUpdated": "10/29/2019",
+							"assetUri": "",
+							"fallbackAssetUri": "fallbackAssetUri",
```

Comment:
Not 100% if these values are correct, I copied them from the agent extension.

Reply:
You need to update the link to point to your VSIX. We can host that for you on our blob storage if you don't have a place yourself.

Reply:
But what about the assetUri and fallBackAssetUri values. Are those correct?

Reply:
And, yes, if you could host the VSIX that would be fantastic! I will send you it through email

Reply:
Yes, those values are fine for assetUri and fallBackAssetUri